README.md

```diff
@@ -1,14 +1,34 @@
 # Coverage Bookend
 [![Version][npm-image]][npm-url] ![Downloads][downloads-image] [![Build Status][status-image]][status-url] [![Open Issues][issues-image]][issues-url] [![Dependency Status][daviddm-image]][daviddm-url] ![License][license-image]
 
-> sd.cd bookend for uploading code coverage reports and/or test results
+> Screwdriver bookend for uploading code coverage reports and/or test results
 
 ## Usage
 
 ```bash
-npm install screwdriver-screwdriver-coverage-bookend
+npm install screwdriver-coverage-bookend
 ```
 
+## Constructor
+This bookend will try to register a Screwdriver coverage npm module based on your configuration provided.
+
+### Example
+
+```yaml
+coverage:
+    plugin: sonar
+    sonar:
+        config:
```

Comment:
Let's take off this `config:` layer here. Based on the logic here https://github.com/screwdriver-cd/coverage-bookend/blob/fc7a37b587d94815e0b2dc1630791dec04d135ae/index.js#L33, it assumes whatever under sonar will be the config object already.

Reply:
Agreed.
I was thinking about that earlier. Because it would be weird in the sonar plugin since it'll have to call `config.config`